packages/odyssey-react/src/components/TextInput/TextInput.test.tsx

```diff
@@ -45,6 +45,25 @@ describe("TextInput", () => {
     expect(screen.getByRole(textBox, { name: label })).toBeVisible();
   });
 
+  it("renders an aria-describedby attribute when an error is present", () => {
+    const error = "oops";
+    render(<TextInput label={label} error={error} />);
+
+    const errorEl = screen.getByText(error);
+    expect(screen.getByRole(textBox, { name: label })).toHaveAttribute(
+      "aria-describedby",
+      errorEl.id
+    );
+  });
+
+  it("does not render an aria-describedby attribute when no error is present", () => {
+    const { container } = render(<TextInput label={label} />);
+
+    const inputElem = container.querySelector("input");
+    const ariaDescribedById = inputElem?.hasAttribute("aria-describedby");
+    expect(ariaDescribedById).toBe(false);
```

Comment:
```suggestion
    render(<TextInput label={label} />);
    expect(screen.getByRole(textBox, { name: label }).).not.toHaveAttribute("aria-describedby");
```

Does this work? It gives us more conventional test code here!